CacheWarmer/HydratorCacheWarmer.php

```diff
@@ -44,6 +45,9 @@ public function isOptional()
 
     public function warmUp($cacheDir)
     {
+        // register custom types during cache warm up to avoid errors while generating hydrator classes
+        $this->registerTypes();
```

Comment:
Why is the logic from the bundle's `boot` method duplicated here? Shouldn't the bundle be booted, thus types being registered?

Reply:
I initially only registered the types in the Bundle::boot()-Method but that resulted in errors during container build time. Only registering during build time obviously doesn't work as that code isn't executed in production. Result of that situation is some code duplication, namely the registerTypes() functions in the Bundle and the HydratorCacheWarmer classes. If someone has a better solution I'm happy to change that.

Reply:
> Shouldn't the bundle be booted, thus types being registered?

Thats what I initally expected aswell.